src/main/resources/templates/project-view.ftl

```diff
@@ -33,6 +33,25 @@
             </form>
         [/#if]
     </div>
+[#elseif branch?? && branch.isBehind()]
```

Comment:
Maybe we should rewrite the if statement so that it is more clear that at this point, the branch is not ahead as well.